src/sections/Art/index.js

```diff
@@ -1,10 +1,50 @@
 import { useEffect, useState } from "react"
 
 function ArtsSection() {
+  
+  // const [artList, setartList] = useState("");
+  const [data, setData] = useState('');
```

Comment:
Be careful with your default values. `data` is going to be an object after you fill it with what you get from the API but you initialized it as a string. Best to keep with the same types, so this should be `const [data, setData] = useState({});` (an empty object)